test/e2e_env/multizone/multizone_suite_test.go

```diff
@@ -0,0 +1,178 @@
+package auth_test
+
+import (
+	"encoding/json"
+	"sync"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma/pkg/config/core"
+	"github.com/kumahq/kuma/pkg/test"
+	"github.com/kumahq/kuma/test/e2e_env/multizone/env"
+	"github.com/kumahq/kuma/test/e2e_env/multizone/trafficpermission"
+	"github.com/kumahq/kuma/test/e2e_env/multizone/trafficroute"
+	. "github.com/kumahq/kuma/test/framework"
+)
+
+func TestE2E(t *testing.T) {
+	test.RunSpecs(t, "E2E Universal Suite")
+}
+
+type State struct {
+	Global    UniversalCPNetworking
+	UniZone1  UniversalCPNetworking
+	UniZone2  UniversalCPNetworking
+	KubeZone1 PortFwd
+	KubeZone2 PortFwd
+}
+
+var _ = SynchronizedBeforeSuite(
+	func() []byte {
+		env.Global = NewUniversalCluster(NewTestingT(), Kuma3, Silent)
+		E2EDeferCleanup(env.Global.DismissCluster) // clean up any containers if needed
+		Expect(env.Global.Install(Kuma(core.Global,
+			WithEnv("KUMA_STORE_UNSAFE_DELETE", "true"),
+		))).To(Succeed())
+
+		wg := sync.WaitGroup{}
+		wg.Add(4)
+
+		env.KubeZone1 = NewK8sCluster(NewTestingT(), Kuma1, Verbose)
+		go func() {
+			Expect(env.KubeZone1.Install(Kuma(core.Zone,
+				WithEnv("KUMA_STORE_UNSAFE_DELETE", "true"),
+				WithIngress(),
+				WithEgress(),
+				WithGlobalAddress(env.Global.GetKuma().GetKDSServerAddress()),
+			))).To(Succeed())
+			wg.Done()
+		}()
+
+		env.KubeZone2 = NewK8sCluster(NewTestingT(), Kuma2, Verbose)
+		go func() {
+			Expect(env.KubeZone2.Install(Kuma(core.Zone,
+				WithEnv("KUMA_STORE_UNSAFE_DELETE", "true"),
+				WithIngress(),
+				// WithEgress(),
```

Comment:
did you leave it intentionally?

Reply:
That was commented because there was a bug, but it's fixed in master now. Restored